app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/DateInfos.kt

```diff
@@ -0,0 +1,56 @@
+package nerd.tuxmobil.fahrplan.congress.models
+
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+import info.metadude.android.eventfahrplan.commons.temporal.Moment.Companion.now
+
+import java.util.ArrayList
+
+class DateInfos : ArrayList<DateInfo>() {
+    fun sameDay(timestamp: Moment, sessionListDay: Int): Boolean {
+        val currentDate = timestamp.startOfDay()
+        return this.find {
+            (dayIndex, date) ->
+                dayIndex == sessionListDay && date == currentDate
+        } != null
+    }
+
+    /**
+     * Returns the index of today.
+     *
+     * @return dayIndex if found, [DateInfo.DAY_INDEX_NOT_FOUND] otherwise.
+     */
```

Comment:
KDoc: Please merge like this `Returns the index of today if found, [DateInfo.DAY_INDEX_NOT_FOUND] otherwise.`